Cargo.toml

```diff
@@ -30,3 +30,8 @@ clap-verbosity-flag = "2.2.2"
 pretty_assertions = "1.4.1"
 url = "2.5.4"
 chrono = "0.4.39"
+serde_yaml = "0.9.32"
```

Comment:
Unfortunately, `serde-yaml` is no longer maintained and we shouldn't be introducing a dependency on it.

It appears that [`serde-yaml-ng`](https://crates.io/crates/serde_yaml_ng) is a fork from an earnest contributor seeking to maintain the original `serde-yaml`.